Do not process the user inputrc, since it might affect the
communication between PLECS and Octave.

--- a/scripts/startup/version-rcfile	2019-11-25 16:04:02.000000000 +0100
+++ b/scripts/startup/version-rcfile	2019-11-25 16:04:28.000000000 +0100
@@ -11,15 +11,15 @@
                                   __octave_config_info__ ("startupfiledir"),
                                   filesep, "inputrc"));
 
-## Re-read user's personal inputrc to give precedence over Octave's default.
-inputrc = getenv ("INPUTRC");
-if (isempty (inputrc) && exist ("~/.inputrc", "file"))
-  inputrc = "~/.inputrc";
-endif
-if (! isempty (inputrc))
-  readline_read_init_file (inputrc);
-endif
-clear ("inputrc");
+### Re-read user's personal inputrc to give precedence over Octave's default.
+#inputrc = getenv ("INPUTRC");
+#if (isempty (inputrc) && exist ("~/.inputrc", "file"))
+#  inputrc = "~/.inputrc";
+#endif
+#if (! isempty (inputrc))
+#  readline_read_init_file (inputrc);
+#endif
+#clear ("inputrc");
 
 ## Configure LESS pager if present
 if (strcmp (PAGER (), "less") && isempty (getenv ("LESS")))
